_content/monographs/can-humanity-change_krishnamurti-jiddu.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Can Humanity Change?"
+authors:
+  - "Jiddu Krishnamurti"
+editor: "David Skitt"
+drive_links:
+  - "https://drive.google.com/file/d/14d9oy9DQyjqZ3j26LrO5jRcYOiIV1NVF/view?usp=drive_link"
```

Comment:
Oh that's very strange... I'm pretty sure this site (or one very similar) used to have this book available for free download: https://www.jkrishnamurti.org/jksearch?keyword=&content_type=16617
Seems perhaps it was removed for copyright reasons?  In which case, probably best to remove this GDrive link and move `course: dialogue` down to a tag